src/Microsoft.ML.Data/Evaluators/Metrics/PrecisionRecallMetrics.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Data
+{
+    /// <summary>
+    /// This class represents one data point on Precision-Recall curve.
+    /// </summary>
+    public sealed class PrecisionRecallMetrics
```

Comment:
>PrecisionRecallMetrics [](start = 24, length = 22)

This appears to be specific to binary PR, yet the name is generic (you use binaryevaluator.)  
If it is for binary, don't you need to also supply positive/negative {recall/precistion|} metrics?
If it  multiclass, you would need a pr matrix afaik

Reply:
changed name to reflect its not a metrics but data point for binary classifications

---
In reply to: [268281758](https://github.com/dotnet/machinelearning/pull/3039#discussion_r268281758) [](ancestors = 268281758)